### ISSUE TITLE
Architecture issue on Raspberry Pi

### ISSUE THREAD

**Issue by QuinnyPig** [browserless/browserless#3803]
There's something odd about the arm64 images being built and pushed to Docker hub.

On a 64-bit Raspian install on a Raspberry Pi 4:

`docker run  --platform linux/arm64 browserless/base` works correctly.
`docker run  --platform linux/arm64 browserless/chrome` throws:
exec ./start.sh: exec format error

linux/arm64 and linux/arm/v8 are supposedly identical architectures; am I missing something obvious here?

**Comment by joelgriffith** [comment]
Have you tried it on V2? Just want to verify the version you're on, which looks like V2 but wanted to check

**Comment by jeayalar** [comment]
Hi @joelgriffith!

I'm having the same issue as @QuinnyPig, although not in a Raspberry Pi, I do use the browserless/chrome linux/arm64 image on an Oracle free tier server. I've used the following docker compose file to pull the image:

```
version: "3.8"

networks:
  browserless:

services:
  browserless:
    container_name: browserless
    environment:
      TOKEN: ${BROWSERLESS_TOKEN}
      EXIT_ON_HEALTH_FAILURE: true
      PRE_REQUEST_HEALTH_CHECK: true
    hostname: browserless
    image: browserless/chrome:latest
    networks:
      - browserless
    ports:
      - 127.0.0.1:9985:3000
    restart: always
```

Similar to @QuinnyPig, my logs are filled with `exec ./start.sh: exec format error`.

```
docker logs browserless --follow

exec ./start.sh: exec format error
exec ./start.sh: exec format error
exec ./start.sh: exec format error
exec ./start.sh: exec format error
exec ./start.sh: exec format error
```

Let me know if you need me to provide more information.

**Comment by QuinnyPig** [comment]
@joelgriffith Indeed; this is v2. `latest` as of yesterday.

**Comment by emc2314** [comment]
The latest docker image of browserless/chrome for linux/arm64 is actually AMD64...
@joelgriffith published wrong architecture files to dockerhub.

**Comment by addicted-ai** [comment]
changed tag of the existing image (working) for now & used that tag in compose file

**Comment by joelgriffith** [comment]
Yes, for legacy versions we were facing issues with buildx building out the arm64 as well as amd64. amd64 is the most important of the two since we utilize it in our cloud environment and elsewhere.

I'm fairly certain the issues is in our `v1` branch here if anyone wants to make a PR since I do not currently have a Pi: https://github.com/browserless/browserless/blob/v1/start.sh

**Comment by joelgriffith** [comment]
@QuinnyPig sorry about that, should have mentioned it! V1 isn't actively being working on, which is the docker hub images. V2, however, _is_ actively being worked on and improved and is available here: https://github.com/orgs/browserless/packages (Chrome, FireFox and Webkit are probably what are the most important to look at).

Currently working on versioned tags atm, so should have something there for y'all shortly.

**Comment by joelgriffith** [comment]
Going to close this as V2 seems to be function which is the direction we're going anyways.

**Comment by mezakos** [comment]
Hi,
So what's the solution for this issue currently? What tag should I use to be able to use the image on arm64?

**Comment by emc2314** [comment]
> Hi, So what's the solution for this issue currently? What tag should I use to be able to use the image on arm64?

TLDR: Use ```docker pull ghcr.io/browserless/chrome:latest```
